2016/10/22 03:47:51 [error] [php] include(UtilManager.php): failed to open stream: No such file or directory (/opt/lampp/htdocs/yii/framework/YiiBase.php:427)
Stack trace:
#0 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(36): spl_autoload_call()
#1 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#3 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#4 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#5 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#6 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#7 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#8 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (36)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/22 05:22:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index".' in /opt/lampp/htdocs/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('index')
#1 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/report/index.php?r=index
---
2016/10/22 05:25:30 [error] [exception.CException] exception 'CException' with message 'Property "User.finish" is not defined.' in /opt/lampp/htdocs/yii/framework/base/CComponent.php:173
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(161): CComponent->__set('finish', 1)
#1 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(60): CActiveRecord->__set('finish', 1)
#2 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(33): UtilManager->updateUser('1', 1)
#3 [internal function]: SiteController->actionAddReport('1', '1', '4455')
#4 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#5 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#6 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addReport')
#10 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addReport')
#11 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#12 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/report/index.php?r=site/addReport&user=1&proj=1&detail=4455
---
2016/10/22 11:35:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "addReport".' in /opt/lampp/htdocs/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('addReport')
#1 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/report/addReport?user=3&proj=1&detail=%E4%BB%8A%E5%A4%A9%E7%8E%A9%E5%BE%97%E5%A5%BD
---
2016/10/22 11:35:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "addReport".' in /opt/lampp/htdocs/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('addReport')
#1 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/report/addReport?user=2&proj=1&detail=%E4%BD%A0%E7%9A%84%E5%B7%A5%E4%BD%9C&_=1477107350768
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/22 11:36:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "addReport".' in /opt/lampp/htdocs/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('addReport')
#1 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/report/addReport?user=2&proj=1&detail=%E4%BD%A0%E7%9A%84%E5%B7%A5%E4%BD%9C&_=1477107378636
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/22 11:38:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "addReport".' in /opt/lampp/htdocs/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('addReport')
#1 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/report/addReport?user=2&proj=1&detail=dssdf&_=1477107496991
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/22 11:38:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "addReport".' in /opt/lampp/htdocs/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('addReport')
#1 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/report/addReport?user=2&proj=1&detail=dssdf
---
2016/10/22 15:49:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] 拒绝连接' in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /opt/lampp/htdocs/report/protected/models/Project.php(16): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(2361): Project->getDbConnection()
#6 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Project))
#7 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(79): CActiveRecord->getMetaData()
#8 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(54): CActiveRecord->__construct()
#9 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(41): UtilManager->addProj('?????????', '3455', '1018', '3')
#10 [internal function]: SiteController->actionAddProject('?????????', '3455', '1018', '3')
#11 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#12 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#13 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#14 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#15 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addproject')
#17 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addProject')
#18 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#19 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/report/site/addProject?name=%E9%BB%91%E8%83%A1%E6%A4%92&jira=3455&plan=1018&owner_id=3&_=1477122569198
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/22 15:49:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] 拒绝连接' in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /opt/lampp/htdocs/report/protected/models/Project.php(16): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(2361): Project->getDbConnection()
#6 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Project))
#7 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(79): CActiveRecord->getMetaData()
#8 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(54): CActiveRecord->__construct()
#9 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(41): UtilManager->addProj('?????????', '3455', '1018', '3')
#10 [internal function]: SiteController->actionAddProject('?????????', '3455', '1018', '3')
#11 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#12 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#13 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#14 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#15 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addproject')
#17 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addProject')
#18 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#19 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/report/site/addProject?name=%E9%BB%91%E8%83%A1%E6%A4%92&jira=3455&plan=1018&owner_id=3&_=1477122577293
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/22 15:49:41 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] 拒绝连接' in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /opt/lampp/htdocs/report/protected/models/User.php(16): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(667): User->getDbConnection()
#6 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(31): CActiveRecord->findAll(Object(CDbCriteria))
#8 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(55): UtilManager->getAllUsers()
#9 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#10 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('')
#14 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#15 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#16 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/report/
---
2016/10/22 21:38:45 [error] [exception.CException] exception 'CException' with message 'Property "Report.owner_id" is not defined.' in /opt/lampp/htdocs/yii/framework/base/CComponent.php:130
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('owner_id')
#1 /opt/lampp/htdocs/yii/framework/base/CModel.php(596): CActiveRecord->__get('owner_id')
#2 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(94): CModel->offsetGet('owner_id')
#3 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#4 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('')
#8 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#9 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/report/
---
2016/10/22 22:02:35 [error] [php] strstr() expects parameter 1 to be string, array given (/opt/lampp/htdocs/report/protected/controllers/SiteController.php:106)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#3 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#4 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#5 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (106)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/22 22:03:47 [error] [php] in_array() expects parameter 2 to be array, string given (/opt/lampp/htdocs/report/protected/controllers/SiteController.php:106)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#3 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#4 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#5 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (106)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/22 22:03:48 [error] [php] in_array() expects parameter 2 to be array, string given (/opt/lampp/htdocs/report/protected/controllers/SiteController.php:106)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#3 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#4 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#5 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (106)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/22 22:04:15 [error] [php] Undefined variable: projectUsers (/opt/lampp/htdocs/report/protected/controllers/SiteController.php:113)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#2 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#3 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#5 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (113)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/23 00:41:53 [error] [php] Use of undefined constant keys - assumed 'keys' (/opt/lampp/htdocs/report/protected/controllers/SiteController.php:43)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#3 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#4 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#5 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#6 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#7 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/site/addReport?user=2&proj=1&detail=445566&_=1477154513720
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (43)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/23 00:43:19 [error] [php] Use of undefined constant keys - assumed 'keys' (/opt/lampp/htdocs/report/protected/controllers/SiteController.php:43)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#3 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#4 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#5 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#6 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#7 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/site/addReport?user=2&proj=1&detail=445566&_=1477154599404
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (43)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/23 00:43:32 [error] [php] Use of undefined constant keys - assumed 'keys' (/opt/lampp/htdocs/report/protected/controllers/SiteController.php:43)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#3 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#4 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#5 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#6 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#7 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/site/addReport?user=3&proj=1&detail=1111&_=1477154612628
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (43)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/23 00:44:23 [error] [php] Use of undefined constant keys - assumed 'keys' (/opt/lampp/htdocs/report/protected/controllers/SiteController.php:43)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#3 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#4 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#5 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#6 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#7 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/site/addReport?user=3&proj=1&detail=1111
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (43)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/23 13:23:42 [error] [php] Object of class User could not be converted to string (/opt/lampp/htdocs/report/protected/views/site/index.php:255)
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CBaseController.php(95): SiteController->renderInternal()
#1 /opt/lampp/htdocs/yii/framework/web/CController.php(869): SiteController->renderFile()
#2 /opt/lampp/htdocs/yii/framework/web/CController.php(782): SiteController->renderPartial()
#3 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(148): SiteController->render()
#4 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#5 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#6 /opt/lampp/htdocs/yii/framework/web/CController.php(286): SiteController->runAction()
#7 /opt/lampp/htdocs/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#8 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): SiteController->run()
#9 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#10 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#11 /opt/lampp/htdocs/report/index.php(13): CWebApplication->run()
REQUEST_URI=/report/
in /opt/lampp/htdocs/report/protected/views/site/index.php (255)
in /opt/lampp/htdocs/report/protected/controllers/SiteController.php (148)
in /opt/lampp/htdocs/report/index.php (13)
2016/10/23 19:37:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/index.js".' in /opt/lampp/htdocs/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('css/index.js')
#1 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/report/css/index.js
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/23 19:52:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/images/bg_header.jpg".' in /opt/lampp/htdocs/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('css/images/bg_h...')
#1 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/report/css/images/bg_header.jpg
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/23 21:40:55 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] 拒绝连接' in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /opt/lampp/htdocs/report/protected/models/Report.php(16): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(2361): Report->getDbConnection()
#6 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Report))
#7 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(79): CActiveRecord->getMetaData()
#8 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(96): CActiveRecord->__construct()
#9 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(31): UtilManager->addReport('1', '14', '????????????')
#10 [internal function]: SiteController->actionAddReport('1', '14', '????????????')
#11 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#12 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#13 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#14 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#15 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addreport')
#17 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addReport')
#18 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#19 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/report/site/addReport?user=1&proj=14&detail=%E6%9B%B4%E5%90%8E%E6%82%94%E8%BF%87&_=1477230055782
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/23 21:41:00 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] 拒绝连接' in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /opt/lampp/htdocs/report/protected/models/Report.php(16): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(2361): Report->getDbConnection()
#6 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Report))
#7 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(79): CActiveRecord->getMetaData()
#8 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(96): CActiveRecord->__construct()
#9 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(31): UtilManager->addReport('1', '14', '????????????')
#10 [internal function]: SiteController->actionAddReport('1', '14', '????????????')
#11 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#12 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#13 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#14 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#15 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addreport')
#17 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addReport')
#18 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#19 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/report/site/addReport?user=1&proj=14&detail=%E6%9B%B4%E5%90%8E%E6%82%94%E8%BF%87&_=1477230060758
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/23 21:41:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] 拒绝连接' in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /opt/lampp/htdocs/report/protected/models/Report.php(16): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(2361): Report->getDbConnection()
#6 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Report))
#7 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(79): CActiveRecord->getMetaData()
#8 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(96): CActiveRecord->__construct()
#9 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(31): UtilManager->addReport('1', '14', '????????????')
#10 [internal function]: SiteController->actionAddReport('1', '14', '????????????')
#11 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#12 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#13 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#14 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#15 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addreport')
#17 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addReport')
#18 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#19 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/report/site/addReport?user=1&proj=14&detail=%E6%9B%B4%E5%90%8E%E6%82%94%E8%BF%87&_=1477230069100
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/23 21:41:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] 拒绝连接' in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /opt/lampp/htdocs/report/protected/models/User.php(16): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(667): User->getDbConnection()
#6 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(52): CActiveRecord->findAll(Object(CDbCriteria))
#8 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(139): UtilManager->getAllUsers()
#9 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#10 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('')
#14 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#15 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#16 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/report/
---
2016/10/23 21:41:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] 拒绝连接' in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /opt/lampp/htdocs/report/protected/models/User.php(16): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(667): User->getDbConnection()
#6 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(52): CActiveRecord->findAll(Object(CDbCriteria))
#8 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(139): UtilManager->getAllUsers()
#9 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#10 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('')
#14 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#15 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#16 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/report/
---
2016/10/23 23:52:27 [error] [php] include(UtilManager.php): failed to open stream: No such file or directory (/opt/lampp/htdocs/yii/framework/YiiBase.php:427)
Stack trace:
#0 /opt/lampp/htdocs/report/protected/commands/CodeCommand.php(8): spl_autoload_call()
#1 unknown(0): CodeCommand->actionCallUser()
#2 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs()
#3 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CodeCommand->run()
#4 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run()
#5 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CConsoleApplication->processRequest()
#6 /opt/lampp/htdocs/report/protected/commands/Crons.php(6): CConsoleApplication->run()

2016/10/23 23:56:20 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'call = 0  id ASC limit 0,1' at line 1. The SQL statement executed was: SELECT id,name,email FROM  `user`  WHERE  finished = 0 and call = 0  id ASC limit 0,1.
2016/10/23 23:56:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'call = 0  id ASC limit 0,1' at line 1' in /opt/lampp/htdocs/yii/framework/db/CDbCommand.php:543
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1581): CDbCommand->queryAll()
#2 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(14): CActiveRecord->findAllBySql('SELECT id,name,...')
#3 /opt/lampp/htdocs/report/protected/commands/CodeCommand.php(9): UtilManager->getCallUser()
#4 [internal function]: CodeCommand->actionCallUser()
#5 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(CodeCommand), Array)
#6 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CConsoleCommand->run(Array)
#7 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run(Array)
#8 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CConsoleApplication->processRequest()
#9 /opt/lampp/htdocs/report/protected/commands/Crons.php(6): CApplication->run()
#10 {main}
---
2016/10/23 23:57:18 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'call = 0 order by id ASC limit 0,1' at line 1. The SQL statement executed was: SELECT id,name,email FROM  `user`  WHERE  finished = 0 and call = 0 order by id ASC limit 0,1.
2016/10/23 23:57:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'call = 0 order by id ASC limit 0,1' at line 1' in /opt/lampp/htdocs/yii/framework/db/CDbCommand.php:543
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1581): CDbCommand->queryAll()
#2 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(14): CActiveRecord->findAllBySql('SELECT id,name,...')
#3 /opt/lampp/htdocs/report/protected/commands/CodeCommand.php(9): UtilManager->getCallUser()
#4 [internal function]: CodeCommand->actionCallUser()
#5 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(CodeCommand), Array)
#6 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CConsoleCommand->run(Array)
#7 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run(Array)
#8 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CConsoleApplication->processRequest()
#9 /opt/lampp/htdocs/report/protected/commands/Crons.php(6): CApplication->run()
#10 {main}
---
2016/10/23 23:59:59 [error] [exception.CDbException] exception 'CDbException' with message 'No columns are being updated for table "user".' in /opt/lampp/htdocs/yii/framework/db/schema/CDbCommandBuilder.php:392
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1765): CDbCommandBuilder->createUpdateCommand(Object(CMysqlTableSchema), Array, Object(CDbCriteria))
#1 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(18): CActiveRecord->updateAll(Array, 'id=:id', Array)
#2 /opt/lampp/htdocs/report/protected/commands/CodeCommand.php(9): UtilManager->getCallUser()
#3 [internal function]: CodeCommand->actionCallUser()
#4 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(CodeCommand), Array)
#5 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CConsoleCommand->run(Array)
#6 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run(Array)
#7 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CConsoleApplication->processRequest()
#8 /opt/lampp/htdocs/report/protected/commands/Crons.php(6): CApplication->run()
#9 {main}
---
2016/10/24 08:44:22 [error] [exception.CDbException] exception 'CDbException' with message 'Table "user" does not have a column named "jira".' in /opt/lampp/htdocs/yii/framework/db/schema/CDbCommandBuilder.php:694
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1518): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(115): CActiveRecord->findByAttributes(Array)
#2 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(53): UtilManager->getProjByJira('22')
#3 [internal function]: SiteController->actionAddProject('3243', '22', '10.17', '1')
#4 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#5 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#6 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addproject')
#10 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addProject')
#11 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#12 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/report/site/addProject?name=3243&jira=22&plan=10.17&owner_id=1&_=1477269863039
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/24 08:44:30 [error] [exception.CDbException] exception 'CDbException' with message 'Table "user" does not have a column named "jira".' in /opt/lampp/htdocs/yii/framework/db/schema/CDbCommandBuilder.php:694
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1518): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(115): CActiveRecord->findByAttributes(Array)
#2 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(53): UtilManager->getProjByJira('22')
#3 [internal function]: SiteController->actionAddProject('3243', '22', '10.17', '1')
#4 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#5 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#6 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addproject')
#10 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addProject')
#11 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#12 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/report/site/addProject?name=3243&jira=22&plan=10.17&owner_id=1&_=1477269870857
HTTP_REFERER=http://115.29.237.213/report/
---
2016/10/24 08:44:46 [error] [exception.CDbException] exception 'CDbException' with message 'Table "user" does not have a column named "jira".' in /opt/lampp/htdocs/yii/framework/db/schema/CDbCommandBuilder.php:694
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/ar/CActiveRecord.php(1518): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 /opt/lampp/htdocs/report/protected/managers/UtilManager.php(115): CActiveRecord->findByAttributes(Array)
#2 /opt/lampp/htdocs/report/protected/controllers/SiteController.php(53): UtilManager->getProjByJira('22')
#3 [internal function]: SiteController->actionAddProject('3243', '22', '10.17', '1')
#4 /opt/lampp/htdocs/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#5 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#6 /opt/lampp/htdocs/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /opt/lampp/htdocs/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /opt/lampp/htdocs/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(282): CController->run('addproject')
#10 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/addProject')
#11 /opt/lampp/htdocs/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#12 /opt/lampp/htdocs/report/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/report/site/addProject?name=3243&jira=22&plan=10.17&owner_id=1&_=1477269863039
---
